src/commonTest/kotlin/exchange.dydx.abacus/app/manager/v2/V4TransactionTests.kt

```diff
@@ -156,7 +156,8 @@ class V4TransactionTests : NetworkTests() {
         subaccountSupervisor?.commitPlaceOrder(0, transactionCallback)
         assertTransactionQueueStarted()
         tradeInput(false, "0.02")
-        subaccountSupervisor?.commitPlaceOrder(0, transactionCallback)
+        val placeOrderPayload = subaccountSupervisor?.commitPlaceOrder(0, transactionCallback)
+        assertEquals("size.size", placeOrderPayload?.sizeInput)
```

Comment:
nice

Reply:
wait, is `sizeInput` supposed to equal `size.size`? or do we need to `parser.value` that